### ISSUE TITLE
a11y: Dropdown child/parent relationship not correct

### ISSUE THREAD

**Issue by jgage176** [microsoft/fluentui#20389]
### Environment Information

- **Package version(s)**: 8.37.4
- **Browser and OS versions**: Win 10/edge 94.0.982.2

### Describe the issue:

Expand out the first dropdown (uncontrolled or even controlled, maybe any of them) and run Accessibility Insights:

Ensures elements with an ARIA role that require child roles contain them (aria-required-children - https://accessibilityinsights.io/info-examples/web/aria-required-children)

Snippet: <div id="Dropdown292-list" class="ms-FocusZone css-64 ms-Dropdown-items dropdownItems-99" role="listbox" aria-label="Control Views" data-focuszone-id="FocusZone324">

![image](https://user-images.githubusercontent.com/29739628/139136717-ce4cb32b-8c3f-42ee-817c-8e05cbb7abf5.png)

I'm not sure if it is a hierarchy issue, but the parent "listbox" is a peer to the "option" items. I wasn't able to hack the dom and get this issue to go away by dragging them, so it may be a different issue.

### Please provide a reproduction of the issue in a codepen:

https://fabricweb.z5.web.core.windows.net/oufr/8.37.4/#/examples/dropdown

#### Actual behavior:

Fails accessibility insights

#### Expected behavior:

Passes accessibility insights

### Documentation describing expected behavior

More information about aria-required-children (https://accessibilityinsights.io/info-examples/web/aria-required-children)

WCAG 1.3.1

**Comment by gouttierre** [comment]
@jgage176  - Thanks for filing this issue with us. To set expectations the developers will review this issue once capacity allows. 

@smhigley - I know you looking at Dropdown atm. Would you be able to confirm if this is a regression, or if this behavior is an issue?  
cc @khmakoto

**Comment by smhigley** [comment]
This was due to a past bug in axe-core, where it was not correctly processing option groups as intermediate children between the listbox and options. This is now fixed, so both axe and accessibility insights no longer report the error.
